### PR TITLE
[Release] Version 1.0.3 - Updated FB iOS SDK.

### PR DESCRIPTION
**Pull request by 1abhishekpandey**
## Description of the change

> Updated the Facebook App Event iOS SDK to version `13.2.0`.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: Rudder-Facebook/Classes/RudderFacebookIntegration.m [hunk 1]
```diff
@@ -67,20 +67,23 @@ - (void) processRuderEvent: (nonnull RSMessage *) message {
...
                 
             }
             else {
-                [FBSDKAppEvents logEvent:truncatedEvent
+                [FBSDKAppEvents.shared logEvent:truncatedEvent
                               parameters:message.properties];
             }
+            [FBSDKAppEvents.shared logEvent: FBSDKAppEventNameAddedToCart
+                    valueToSum: 100.0
+                    parameters: message.properties];
```
Comment (desusai7) [inline Rudder-Facebook/Classes/RudderFacebookIntegration.m]: Why are we doing this ?

Comment (1abhishekpandey) [reply Rudder-Facebook/Classes/RudderFacebookIntegration.m]: I'll remove this. This need not to be present.

**Review by desusai7 — APPROVED** [review]

File: Rudder-Facebook.podspec [hunk 2]
```diff
@@ -10,11 +10,11 @@ Pod::Spec.new do |s|
...
     s.license          = { :type => "Apache", :file => "LICENSE" }
     s.author           = { 'Rudderlabs' => 'arnab@rudderlabs.com' }
-    s.source           = { :git => 'https://github.com/rudderlabs/rudder-integration-facebook-ios.git', :tag => 'v1.0.2' }
+    s.source           = { :git => 'https://github.com/rudderlabs/rudder-integration-facebook-ios.git', :tag => 'v1.0.3' }
     s.platform         = :ios, "9.0"
 
     s.source_files = 'Rudder-Facebook/Classes/**/*'
 
     s.dependency 'Rudder', '~> 1.0'
-    s.dependency 'FBSDKCoreKit', '12.0.2'
```
Comment (pallabmaiti) [inline Rudder-Facebook.podspec:19]: Let's make it flexible.

Comment (pallabmaiti) [inline Rudder-Facebook.podspec:19]: Let's make it flexible from 13.2.0

**Review by pallabmaiti — APPROVED** [review]

**Pull request merged**